ndc-client/src/models.rs

```diff
@@ -163,6 +177,16 @@ pub struct AggregateFunctionDefinition {
 }
 // ANCHOR_END: AggregateFunctionDefinition
 
+// ANCHOR: UpdateOperatorDefinition
+/// The definition of an update operator on a scalar type
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize, JsonSchema)]
+#[schemars(title = "Update Operator Definition")]
+pub struct UpdateOperatorDefinition {
+    /// The type of the argument to this operator
+    pub argument_type: Type,
```

Comment:
Thinking again of something like an `increment` operator, does making this `Type` optional make sense? Or there is nicer way to express this?

Reply:
Ignore this, I misunderstood - this is the type of the target column, ie `int`.